addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -1450,16 +1493,21 @@ var SnippetsMenu = Widget.extend({
         this.$snippets.each(function () {
             var $snippet = $(this);
             var $snippetBody = $snippet.find('.oe_snippet_body');
+            const isSanitizeForbidden = $snippet.data('oeForbidSanitize');
+            function filterSanitize($el) {
+                return isSanitizeForbidden ? $el.not('[data-oe-sanitize], [data-oe-sanitize] *') : $el;
+            }
 
             var check = false;
             _.each(self.templateOptions, function (option, k) {
+                k = isSanitizeForbidden ? 'forbidden/' + k : k;
```

Comment:
I am not familiar with this part of the code, could you explain to me why do you need a different cache key when forbidden?